project/ScalafixBuild.scala

```diff
@@ -127,35 +125,30 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
     updateOptions := updateOptions.value.withCachedResolution(true),
     ThisBuild / watchTriggeredMessage := Watch.clearScreenOnTrigger,
     commands += Command.command("save-expect") { s =>
-      "unit/test:runMain scalafix.tests.util.SaveExpect" ::
+      "unit2_13Target2_13/test:runMain scalafix.tests.util.SaveExpect" ::
         s
     },
     commands += Command.command("ci-3") { s =>
-      s"""set testsInput/scalaVersion := "$scala3"""" ::
-        s"""set testsOutput/scalaVersion := "$scala3"""" ::
-        "unit/testOnly scalafix.tests.rule.RuleSuite" :: s
+      "unit2_12Target3/test" ::
+        s
     },
     commands += Command.command("ci-213") { s =>
-      s"""set ThisBuild/scalaVersion := "$scala213"""" ::
-        "unit/test" ::
-        "docs/run" ::
-        "interfaces/doc" ::
+      "unit2_13Target2_13/test" ::
+        "docs2_13/run" ::
+        "interfaces2_13/doc" ::
         testRulesAgainstPreviousScalaVersions(scala213, s)
     },
     commands += Command.command("ci-212") { s =>
-      s"""set ThisBuild/scalaVersion := "$scala212"""" ::
-        "unit/test" ::
+      "unit2_12Target2_12/test" ::
         testRulesAgainstPreviousScalaVersions(scala212, s)
     },
     commands += Command.command("ci-211") { s =>
-      s"""set ThisBuild/scalaVersion := "$scala211"""" ::
-        "unit/test" ::
+      "unit2_11Target2_11/test" ::
         testRulesAgainstPreviousScalaVersions(scala211, s)
     },
     commands += Command.command("ci-213-windows") { s =>
-      s"++$scala213" ::
-        "cli/crossPublishLocalBinTransitive" :: // scalafix.tests.interfaces.ScalafixSuite
-        s"unit/testOnly -- -l scalafix.internal.tests.utils.SkipWindows" ::
+      "publishLocalTransitive" :: // scalafix.tests.interfaces.ScalafixSuite
```

Comment:
`publishLocalTransitive` is only defined in `cli*` projects so running this at the root triggers them through aggregation